From bac48ee16a636897ff4c8e32222833fef7a50c08 Mon Sep 17 00:00:00 2001
From: Calixte Denizet <calixte.denizet@gmail.com>
Date: Wed, 7 Oct 2020 15:59:11 +0200
Subject: [PATCH] Extra stuff for use with PDF.js

---
 quickjs.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/quickjs.c b/quickjs.c
index efc1d54..985bebd 100644
--- a/quickjs.c
+++ b/quickjs.c
@@ -47444,6 +47444,8 @@ static JSValue js_global_unescape(JSContext *ctx, JSValueConst this_val,
     return string_buffer_end(b);
 }
 
+#include "quickjs_extra.c"
+
 /* global object */
 
 static const JSCFunctionListEntry js_global_funcs[] = {
@@ -47467,6 +47469,7 @@ static const JSCFunctionListEntry js_global_funcs[] = {
     //JS_CFUNC_DEF("__date_now", 0, js___date_now ),
     //JS_CFUNC_DEF("__date_getTimezoneOffset", 1, js___date_getTimezoneOffset ),
     //JS_CFUNC_DEF("__date_create", 3, js___date_create ),
+    EXTRA
 };
 
 /* Date */
-- 
2.28.0

