From f7a513c626e9aae5c431ed13bb60838e2a548650 Mon Sep 17 00:00:00 2001
From: Calixte Denizet <calixte.denizet@gmail.com>
Date: Fri, 9 Feb 2024 09:54:34 +0100
Subject: [PATCH] Extra stuff for use with PDF.js

---
 quickjs.c | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/quickjs.c b/quickjs.c
index 4e58a98..b103501 100644
--- a/quickjs.c
+++ b/quickjs.c
@@ -49247,6 +49247,8 @@ static JSValue js_global_unescape(JSContext *ctx, JSValueConst this_val,
     return string_buffer_end(b);
 }
 
+#include "quickjs_extra.c"
+
 /* global object */
 
 static const JSCFunctionListEntry js_global_funcs[] = {
@@ -49264,6 +49266,7 @@ static const JSCFunctionListEntry js_global_funcs[] = {
     JS_PROP_DOUBLE_DEF("Infinity", 1.0 / 0.0, 0 ),
     JS_PROP_DOUBLE_DEF("NaN", NAN, 0 ),
     JS_PROP_UNDEFINED_DEF("undefined", 0 ),
+    EXTRA
 };
 
 /* Date */
@@ -49279,9 +49282,10 @@ static int64_t floor_div(int64_t a, int64_t b) {
     int64_t m = a % b;
     return (a - (m + (m < 0) * b)) / b;
 }
-
+#if 0
 static JSValue js_Date_parse(JSContext *ctx, JSValueConst this_val,
                              int argc, JSValueConst *argv);
+#endif
 
 static __exception int JS_ThisTimeValue(JSContext *ctx, double *valp, JSValueConst this_val)
 {
@@ -49834,7 +49838,7 @@ static int string_get_month(JSString *sp, int *pp, int64_t *pval) {
     *pp += 3;
     return 0;
 }
-
+#if 0
 static JSValue js_Date_parse(JSContext *ctx, JSValueConst this_val,
                              int argc, JSValueConst *argv)
 {
@@ -49997,7 +50001,7 @@ done:
     JS_FreeValue(ctx, s);
     return rv;
 }
-
+#endif
 static JSValue js_Date_now(JSContext *ctx, JSValueConst this_val,
                            int argc, JSValueConst *argv)
 {
-- 
2.25.1

