From dce526f8e5471cf28c646270d71ab3f23d6be822 Mon Sep 17 00:00:00 2001
From: calixteman <calixte.denizet@gmail.com>
Date: Sat, 21 Feb 2026 15:07:37 +0100
Subject: [PATCH] Extra stuff for use with PDF.js

---
 quickjs.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/quickjs.c b/quickjs.c
index fa5a7a9..4b83154 100644
--- a/quickjs.c
+++ b/quickjs.c
@@ -54762,6 +54762,8 @@ static JSValue js_global_unescape(JSContext *ctx, JSValueConst this_val,
     return string_buffer_end(b);
 }
 
+#include "quickjs_extra.c"
+
 /* global object */
 
 static const JSCFunctionListEntry js_global_funcs[] = {
@@ -54781,6 +54783,7 @@ static const JSCFunctionListEntry js_global_funcs[] = {
     JS_PROP_UNDEFINED_DEF("undefined", 0 ),
     JS_PROP_STRING_DEF("[Symbol.toStringTag]", "global", JS_PROP_CONFIGURABLE ),
     JS_CFUNC_DEF("eval", 1, js_global_eval ),
+    EXTRA
 };
 
 /* Date */
@@ -54797,8 +54800,10 @@ static int64_t floor_div(int64_t a, int64_t b) {
     return (a - (m + (m < 0) * b)) / b;
 }
 
+#if 0
 static JSValue js_Date_parse(JSContext *ctx, JSValueConst this_val,
                              int argc, JSValueConst *argv);
+#endif
 
 static __exception int JS_ThisTimeValue(JSContext *ctx, double *valp, JSValueConst this_val)
 {
@@ -55677,6 +55682,7 @@ static BOOL js_date_parse_otherstring(const uint8_t *sp,
     return TRUE;
 }
 
+#if 0
 static JSValue js_Date_parse(JSContext *ctx, JSValueConst this_val,
                              int argc, JSValueConst *argv)
 {
@@ -55726,6 +55732,7 @@ static JSValue js_Date_parse(JSContext *ctx, JSValueConst this_val,
     JS_FreeValue(ctx, s);
     return rv;
 }
+#endif
 
 static JSValue js_Date_now(JSContext *ctx, JSValueConst this_val,
                            int argc, JSValueConst *argv)
-- 
2.53.0

